apps/vscode/command_client.py

```diff
@@ -0,0 +1,196 @@
+import requests
+import time
+import json
+from typing import Any
+from talon import Module, actions, app
+from pathlib import Path
+from tempfile import gettempdir
+
+is_mac = app.platform == "mac"
+
+mod = Module()
+
+
+class NotSet:
+    def __repr__(self):
+        return "<argument not set>"
+
+def run_vscode_command_by_command_palette(command: str):
+    """Execute command via command palette. Preserves the clipboard."""
+    # Clip is noticeably faster than insert
+    if not is_mac:
+        actions.key("ctrl-shift-p")
+    else:
+        actions.key("cmd-shift-p")
```

Comment:
Not sure I follow; can you elaborate?

Reply:
Instead of hardcoding a keyboard shortcut in the middle of your code, prefer using an action like "command_palette()" that is overridden on Mac (instead of internally switching based on app.platform).